packages/vscode-extension/src/commandsTreeViewProvider.ts

```diff
@@ -496,13 +496,37 @@ export class CommandsTreeViewProvider implements vscode.TreeDataProvider<TreeVie
               );
               this.disposableMap.set(subTreeItem.commandId, disposable);
             }
+
+            let tooltip: string | vscode.MarkdownString = treeItem.label;
```

Comment:
If subTreeItem.tooltip equals undefined and treeItem.label exists, then the tooltip of subTreeItem will becomes treeItem.label?

Reply:
Yes, that's true

Reply:
Updated this logic, thanks!